### ISSUE TITLE
Program exits after sleep time

### ISSUE THREAD

**Issue by aNiceGuy69** [SamueleAmato/instaForce#6]
I followed your instructions to install all the required packages etc. Sadly it does not work. After I execute "main.py", cmd exits after I put "900" as sleep time.
Thanks for the help!

**Comment by aNiceGuy69** [comment]
> Sleep time is probably not the problem, are you sure that in the folder where the file "main.py" is there is a text file called "wl.txt"?

yes, I made my own textfile and named it "wl.txt". It is in **C:\Users\matt\instaForce**, or should it maybe be in **C:\Users\matti\instaForce\src\instaForce**?

> --> So try entering **rockyou.txt and not wl.txt** and let me know if it worked out.

I tried that and it threw this error message:

```
Word list: rockyou.txt

Insert sleep time for login [Raccomanded 900(15min)]: 900
Traceback (most recent call last):
  File "C:\Users\matti\instaForce\main.py", line 21, in <module>
    termux()
  File "C:\Users\matti\instaForce\src\termux.py", line 34, in termux
    sub.call("clear")
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 389, in call
    with Popen(*popenargs, **kwargs) as p:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1022, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1491, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by SamueleAmato** [comment]
I think I found the problem, after re-downloaded the repository go to "instaForce/main.py"

You'll see something like:

```
if systemis == "Debian":
    linux()
elif systemis == "Arch":
    arch()
elif systemis == "Termux":
    termux()
elif systemis == "Windows":
    termux()
```

Replace the last function like this:

```
elif systemis == "Windows":
    windows()
```

![1](https://user-images.githubusercontent.com/107519227/203391421-9784628a-d3d1-431c-8b8a-1ee8c06c66fb.png)


![ksnip_20221122-191918](https://user-images.githubusercontent.com/107519227/203391478-01fa712e-6e53-41aa-a97b-7f88af37d341.png)

_(I will fix this in the next version that should be out soon)_

If that doesn't work make sure you have selected the "windows" option in the setup and downloaded the latest version, I tested the program on a vm with win7 and it worked, the last of the options is enable wsl (windows linux subsystem)

**Comment by aNiceGuy69** [comment]
I reinstalled the repository via git clone (I hope that installs the newest version) and changed C:\Users\matti\instaForce\main.py to this:

![image](https://user-images.githubusercontent.com/62745765/203416306-2e99b64e-b392-4487-958c-d3c97943d8a7.png)

Sadly it threw this error:

```
Insert sleep time for login [Raccomanded 900(15min)]: 900
Traceback (most recent call last):
  File "C:\Users\matti\instaForce\main.py", line 21, in <module>
    windows()
  File "C:\Users\matti\instaForce\src\windows.py", line 33, in windows
    sub.call("clear")
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 389, in call
    with Popen(*popenargs, **kwargs) as p:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1022, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\matti\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1491, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

should I maybe use an older verison of python3?
> the last of the options is enable wsl (windows linux subsystem)
I have never done that before, how would I do that exactly?

Thanks for the help! This is awesome!

**Comment by SamueleAmato** [comment]
I released a new update (0.3.0) I added a few things, I also changed the file to run on windows, I noticed that although I wrote the file for windows I forgot to replace the command "clear" which on windows is "cls", now I made these changes;
> try re-downloading the repository and running the setup again.

If it doesn't work this time either the only one is to install WSL and download the directory, then when you run the setup instead of windows you select "linux debian"

wsl tutorial --> https://www.youtube.com/watch?v=av0UQy6g2FA

**Comment by aNiceGuy69** [comment]
Okay I got it running on wsl now. Thanks for the help!
